app/viewmodels/CashStatementViewModel.scala

```diff
@@ -31,29 +31,32 @@ import java.time.LocalDate
 case class CashStatementViewModel(statementsForAllEoris: Seq[CashStatementForEori])
 
 case class CashStatementForEori(eoriHistory: EoriHistory,
-                                currentStatements: Seq[CashStatementByMonth],
-                                requestedStatements: Seq[CashStatementByMonth])
+                                currentStatements: Seq[CashStatementMonthToMonth],
+                                requestedStatements: Seq[CashStatementMonthToMonth])
   extends OrderedByEoriHistory[CashStatementForEori]
 
-case class CashStatementByMonth(date: LocalDate, files: Seq[CashStatementFile] = Seq.empty)
-                               (implicit messages: Messages) extends Ordered[CashStatementByMonth] {
+case class CashStatementMonthToMonth(startDate: LocalDate, endDate: LocalDate, files: Seq[CashStatementFile] = Seq.empty)
+                                    (implicit messages: Messages) extends Ordered[CashStatementMonthToMonth] {
 
   val formattedMonthToMonth: String = files.headOption.map { file =>
-    Formatters.periodAsStartToEndMonth(file.metadata.periodStartMonth, file.metadata.periodEndMonth)
+    Formatters.periodAsStartToEndMonth(startDate.getMonthValue, endDate.getMonthValue)
   }.getOrElse(emptyString)
 
   val cashAccountNumber: Option[String] = files.headOption.flatMap(_.metadata.cashAccountNumber)
-  val formattedMonthYear: String = Formatters.dateAsMonthAndYear(date)
+  val formattedMonthYear: String = Formatters.dateAsMonthAndYear(startDate)
 
   val pdf: Option[CashStatementFile] = files.find(_.fileFormat == Pdf)
   val csv: Option[CashStatementFile] = files.find(_.fileFormat == Csv)
 
-  override def compare(that: CashStatementByMonth): Int = date.compareTo(that.date)
+  override def compare(that: CashStatementMonthToMonth): Int = {
+    val startComparison = this.startDate.compareTo(that.startDate)
```

Comment:
val startComparision gives impression like to start a comparision function. Pls give a bette name .. may be like 'comparisionResult' or something

Reply:
Renamed to `priorityComparison`, hopefully that makes more sense.